src/migrations/20220802074933-session_form.js

```diff
@@ -0,0 +1,128 @@
+const endpoints = require('../constants/endpoints')
+const request = require('request')
+module.exports = {
+	async up(db) {
+		global.migrationMsg = 'Session Form Created'
+
+		const form = await db.collection('forms').findOne({ type: 'session' })
+		if (!form) {
+			const categories = await db
+				.collection('entities')
+				.find(
+					{ type: 'categories', status: 'ACTIVE', deleted: false },
+					{ projection: { value: 1, label: 1, image: 1, _id: 0 } }
+				)
+				.toArray()
+
+			let formsData = {
+				type: 'session',
+				subType: 'sessionForm',
+				action: 'sessionFields',
+				ver: '1.0',
+				data: {
+					templateName: 'defaultTemplate',
+					fields: {
+						controls: [
+							{
+								name: 'title',
+								label: 'Session Title',
+								value: '',
+								class: 'ion-margin',
+								type: 'text',
+								position: 'floating',
+								validators: { required: true },
+							},
+							{
+								name: 'description',
+								label: 'Description',
```

Comment:
@ankitpws value , class,position and validators are the same for all fields, no need to mention in the all the places, clear the code